src/Microsoft.IdentityModel.Tokens.Saml/Saml2/Saml2Serializer.cs

```diff
@@ -1516,7 +1752,7 @@ protected virtual void WriteAdvice(XmlWriter writer, Saml2Advice advice)
         }
 
         /// <summary>
-        /// Writes the &lt;Assertion> element.
+        /// Writes the &lt;Assertion> or &lt;EncryptedAssertion> element.
```

Comment:
>> [](start = 36, length = 1)

consider : gt; ? ... same towards the end of the line..

Reply:
Just being consistent - this pattern (&lt *NODE* >) is used across the library.